internal/hierarchyconfig/validator_test.go

```diff
@@ -17,6 +17,101 @@ import (
 	"sigs.k8s.io/hierarchical-namespaces/internal/objects"
 )
 
+func TestManagedMetaPermissions(t *testing.T) {
+	f := foresttest.Create("-") // a
+	h := &Validator{Forest: f}
+	l := zap.New()
+
+	tests := []struct {
+		name        string
+		labels      []api.MetaKVP
+		annotations []api.MetaKVP
+		allowed     bool
+	}{
+		{name: "managed label", labels: []api.MetaKVP{{Key: "label.com/team"}}, allowed: true},
+		{name: "unmanaged label", labels: []api.MetaKVP{{Key: "kubernetes.io/metadata.name"}}},
+		{name: "managed annotation", annotations: []api.MetaKVP{{Key: "annot.com/log-index"}}, allowed: true},
+		{name: "unmanaged annotation", annotations: []api.MetaKVP{{Key: "openshift.io/sa.scc.uid-range"}}},
+	}
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			// Setup
+			g := NewWithT(t)
+			// In this test we only allow labels/annotations with the defined prefixes
+			err := config.SetManagedMeta([]string{"label\\.com/.*"}, []string{"annot\\.com/.*"})
```

Comment:
I wish we had an alternative to regex here. IMO a prefix- or glob-based pattern would be much more user-friendly here. Prefixed labels/annotations are recommended and very common.

Reply:
Discussed on Slack, maybe add an issue for this? Internally we'll almost certainly stick to regexes but we could have a more user-friendly way to set the command lines (just like, for exceptions, we have the generic `/select` annotation but also `/treeSelect`).

Reply:
Created issue.